google/resource_container_cluster.go

```diff
@@ -94,6 +94,25 @@ func resourceContainerCluster() *schema.Resource {
 				},
 			},
 
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"node_locations": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"num_nodes": {
```

Comment:
This doesn't seem to get used anywhere

Reply:
I see that this is not a new field added to support regional clusters. So this is not relevant to this PR. Removing this. But we'll have to add this later.